swap_meet/electronics.py

```diff
@@ -0,0 +1,10 @@
+from swap_meet.item import Item
+
+
+class Electronics(Item):
+    def __init__(self, condition=0):
+        self.category = "Electronics"
+        self.condition = condition
```

Comment:
Same comment in Clothing can be applied here

Reply:
Done!